rfm-master/com.c

```diff
@@ -371,324 +387,311 @@ static void print_incomplete_mark(int8_t len) {
  *  \note command X.....\n    - X is upcase char as commad name, \n is termination char
  *  \note hex numbers use ONLY lowcase chars, upcase is reserved for commands
  *  \note   V\n - print version information
- *  \note   D\n - print status line 
+ *  \note   D\n - print status line
  *  \note   Yyymmdd\n - set, year yy, month mm, day dd; HEX values!!!
  *  \note   HhhmmSSss\n - set, hour hh, minute mm, second SS, 1/100 second ss; HEX values!!!
- *	
+ *
  ******************************************************************************/
-void COM_commad_parse (void) {
+void COM_commad_parse(void)
+{
 	char c;
+
 	while (COM_requests) {
-        switch(c=COM_getchar()) {
+		switch (c = COM_getchar()) {
 		case 'V':
-			if (COM_getchar()=='\n') print_version();
-			c='\0';
+			if (COM_getchar() == '\n') print_version();
+			c = '\0';
 			break;
 		case 'D':
-			if (COM_getchar()=='\n') COM_print_debug(-1);
-			c='\0';
+			if (COM_getchar() == '\n') COM_print_debug(-1);
+			c = '\0';
 			break;
 		case 'Y':
-			if (COM_hex_parse(3*2,true)!='\0') { break; }
-			RTC_SetDate(com_hex[2],com_hex[1],com_hex[0]);
-            print_s_p(PSTR("OK"));
+			if (COM_hex_parse(3 * 2, true) != '\0') break; RTC_SetDate(com_hex[2], com_hex[1], com_hex[0]);

```

Comment:
Similar formatting problem resulting in ugly one-liner. This is not about empty loop body, but similar failure.